src/librustc_middle/mir/interpret/error.rs

```diff
@@ -467,6 +469,7 @@ impl fmt::Debug for UnsupportedOpInfo {
             NoMirFor(did) => write!(f, "could not load MIR for {:?}", did),
             ReadPointerAsBytes => write!(f, "unable to turn pointer into raw bytes",),
             ReadBytesAsPointer => write!(f, "unable to turn bytes into a pointer"),
+            ThreadLocalStatic(did) => write!(f, "tried to access thread local static {:?}", did),
```

Comment:
I've recently been changing the wording of these from "tried to do thing" to "doing thing". "Try" sounds too much like a `try` method to me which generally safely do nothing when the operation cannot be performed.